js/kubewarden/host_capabilities/kubernetes/kubernetes.ts

```diff
@@ -28,4 +90,27 @@ export namespace Kubernetes {
       throw new Error(`Failed to decode or parse getResource response: ${err}`);
     }
   }
+
+  export function canI(req: CanIRequest): SubjectAccessReviewStatus {
+    let payload: ArrayBuffer;
+    try {
+      payload = new TextEncoder().encode(JSON.stringify(req)).buffer;
+    } catch (err) {
+      throw new Error(`Cannot serialize CanIRequest: ${err}`);
+    }
+
+    let responsePayload: ArrayBuffer;
+    try {
+      responsePayload = HostCall.hostCall('kubewarden', 'kubernetes', 'can_i', payload);
+    } catch (err) {
+      throw new Error(`${err}`);
+    }
+
+    try {
+      const jsonString = new TextDecoder().decode(responsePayload);
+      return JSON.parse(jsonString) as SubjectAccessReviewStatus;
```

Comment:
I also just recalled that `JSON.parse` is not going to do any validation. Does that mean that doing `JSON.parse("123") as SubjectAccessReviewStatus` is not going to raise any error since `123` is a valid JSON object?